sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -48,6 +48,8 @@
 
 TRANSCEIVER_STATUS_TABLE_SW_FIELDS = ["status", "error"]
 
+CMIS_MODULE_TYPES    = ['QSFP-DD', 'QSFP_DD', 'OSFP', 'QSFP+C']
```

Comment:
can we add a new API to detect whether the cable is CMIS? otherwise, we have to update this list when new cables are supported.

Reply:
I have removed this now and added a API to identify CMIS based transceivers using "isinstance" API.